README.md

```diff
@@ -25,3 +23,37 @@ ds = open_dataset(
     branch="my_new_testdata_branch"
 )
 ```
+
+> **Note**
+> The following options only work for branches based on `Ouranosinc/xclim-testdata`, not forks.
+
+If you wish to run the entire `xclim` testing suite locally against your branch, this can be set via an environment variable:
+```shell
+$ export XCLIM_TESTDATA_BRANCH="my_new_testdata_branch"
+
+$ pytest xclim
+# or, alternatively:
+$ tox
+```
+
+If you wish to run the entire `xclim` testing suite on the `Ouranosinc/xclim` GitHub Workflows (CI) against your branch,
+this can be set via an environment variable default in the `.github/workflows/main.yml` workflow configuration:
+```yaml
+env:
+  XCLIM_TESTDATA_BRANCH: my_new_testdata_branch
+```
+
+> **Note**
+> Be aware that modifying this variable to a value other than the latest tagged version of `xclim-testdata`
+> will trigger a GitHub Workflow that will block merging of your Pull Request until changes are effected.
```

Comment:
"latest tagged version of `xclim-testdata`": all tags for certain format of tags only?

Should probably check for "latest of format #.#.#" only and not all tags, leaving other format of tags to be newer but not considered as "latest" tag.